lib/publiccloud/utils.pm

```diff
@@ -172,4 +172,12 @@ sub is_gce() {
     return is_public_cloud && check_var('PUBLIC_CLOUD_PROVIDER', 'GCE');
 }
 
+sub define_secret_variable {
```

Comment:
@foursixnine  this function might be useful for broader audience , I wonder if it make sense to move it into `lib/utils` or maybe some other place which you have in your mind ...

Reply:
@asmorodskyi I think that it could be moved to it's own namespace (say `Utils::Security::Commands` not to pollute `lib/utils`...  but that would be for later in the future, so I guess for now here is ok, once I find a place good enough, I'll come back to this.

Pinging @punkioudi and @michaelgrifalconi as they are working on the leaking story atm